db/test/test_stats.py

```diff
@@ -272,15 +274,17 @@ def add_empty_lowlevel(mbid, lossless, date):
     data_sha256 = str(lossless) + str(mbid) + str(date)
     data_sha256 = data_sha256[:64]
     data_json = "{}"
+    gid_type = 'msid'
     with db.engine.connect() as connection:
         query = text("""
-            INSERT INTO lowlevel (mbid, build_sha1, lossless, submitted)
-                 VALUES (:mbid, :build_sha1, :lossless, :submitted)
+            INSERT INTO lowlevel (gid, build_sha1, lossless, submitted, gid_type)
+                 VALUES (:mbid, :build_sha1, :lossless, :submitted, :gid_type)
               RETURNING id
         """)
         result = connection.execute(query,
                 {"mbid": mbid, "build_sha1": build_sha1,
-                 "lossless": lossless, "submitted": date})
+                 "lossless": lossless, "submitted": date,
+                 "gid_type": gid_type})
```

Comment:
`gid_type` variable is used once here and can be replaced with `GID_TYPE_MSID`, similar to other places.